src/types/index.ts

```diff
@@ -113,6 +135,7 @@ export interface INetwork {
   networkId: string
   tipContractV1: string
   tipContractV2?: string
+  multisigBackend: string
```

Comment:
We have the `backendUrl` so this should be `multisigBackendUrl` (missing `Url`).